lib/cog/adapters/slack/rtm_connector.ex

```diff
@@ -112,6 +114,17 @@ defmodule Cog.Adapters.Slack.RTMConnector do
     end
     handler.()
   end
+  def handle_message(%{type: channel_event, channel: channel}, _slack, state) when channel_event in @channel_events do
+    channel_id = case channel do
+      channel when is_binary(channel) ->
+        channel
+      %{id: id} ->
```

Comment:
There's not really a method to it from what I can tell. Some events give you a map, and others an id. If there was a pattern I'd call it out, but a comment like "Some events give you a map, and others an id" is kinda just reiterating what the code already says.